ticketutil/redmine.py

```diff
@@ -63,21 +63,45 @@ def _verify_project(self, project):
             logger.error("Project {0} is not valid".format(project))
             return False
 
+    def get_ticket_content(self, ticket_id=None):
+        """
+        Queries the Redmine API to get the ticket_content using ticket_id.
+        :param ticket_id: ticket number, if not set self.ticket_id is used.
+        :return: self.request_result: Named tuple containing request status, error_message, url info and
+                 ticket_content.
+        """
+        if ticket_id is None:
+            ticket_id = self.ticket_id
+            if not self.ticket_id:
+                error_message = "No ticket ID associated with ticket object. " \
+                                "Set ticket ID with set_ticket_id(<ticket_id>)"
+                logger.error(error_message)
+                return self.request_result._replace(status='Failure', error_message=error_message)
+        try:
+            r = self.s.get("{0}/{1}.json?include=attachments,journals,watchers".format(self.rest_url, ticket_id))
```

Comment:
According to the API spec, there are other options for the include param: children, attachments, relations, changesets, journals, watchers. Any reason you only included three of them? A couple solutions would be to

1) Define an 'include' parameter to the method for the user to specify these options, which could take in a string or a list.
2) You can include the other options in the request url: children, relations, changesets. Making the whole url:

`"{0}/{1}.json?include=attachments,journals,watchers,children,relations,changesets".format(self.rest_url, ticket_id)`

I tried this out and it seemed to work, even when they weren't valid options for a particular ticket.

Do you prefer either of these solutions?

Reply:
I didn't see the point using all include options, as `attachments,journals,watchers` are the only ones used in code (in methods `add_attachment()`, `add_comment` and `add_watcher`).

I prefer option number two, so I used all include options.